app/controllers/sessions_controller.rb

```diff
@@ -4,7 +4,7 @@
 class SessionsController < ApplicationController
   #protect_from_forgery :except => [:create]
 
-  @@openid_url_base  = "https://publiclab.org/people/"
+  @@openid_url_base  = "localhost:3000/people/"
```

Comment:
This line needs to be removed before the merge. I am having it to test the servers are working properly simultaneously